src/main/ngage/SDL_ngage_main.cpp

```diff
@@ -1,6 +1,5 @@
 /*
-    EPOC version (originally for SDL 1.2) by Hannu Viitala
-    (hannu.j.viitala@mbnet.fi).
+    SDL_psp_main.c, placed in the public domain by Sam Lantinga  3/13/14
```

Comment:
Was this intentional?

Reply:
The SDL 2 adaptation has its origin in a SDL 1.2 port which was explicitly written for a Doom port for the Nokia 9210. A lot has happened since then.

I have mentioned all those involved in the readme and referred to the history of the port. That's a better place for it.

Reply:
`SDL_psp_main.c` looks funny for ngage port (I guess it's result of a copy+paste.) What is meant here is name and author correction.

Reply:
Whoops, I missed that. Definitely not intentional.